lisa/tools/nvmecli.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.tools import Git, Make
+
+
+class Nvmecli(Tool):
+    repo = "https://github.com/linux-nvme/nvme-cli"
+
+    @property
+    def command(self) -> str:
+        return "nvme"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def _install_from_src(self) -> None:
+        posix_os: Posix = cast(Posix, self.node.os)
+        tool_path = self.get_tool_path()
+        self.node.shell.mkdir(tool_path, exist_ok=True)
+        git = self.node.tools[Git]
```

Comment:
Call `os.install_packages([Git, Make])`, it would be faster than call them one by one.